libs/designsystem/src/lib/components/icon/icon-registry.service.spec.ts

```diff
@@ -12,21 +11,6 @@ describe('KirbyIconRegistryService', () => {
     expect(service).toBeTruthy();
   });
 
-  it('should register icons if injected', () => {
-    const consoleWarnSpy = spyOn(console, 'warn');
-    const iconSettings: IconSettings = {
-      icons: [
-        { name: 'name1', svg: 'svg1' },
-        { name: 'name2', svg: 'svg2' },
-      ],
-    };
-    service = new IconRegistryService(iconSettings);
-    expect(service.getIcons()).toEqual(iconSettings.icons);
-    expect(consoleWarnSpy).toHaveBeenCalledWith(
-      'Use of IconSettings is deprecated, use IconRegistryService instead'
-    );
-  });
-
```

Comment:
@mictro, @jkaltoft, @RasmusKjeldgaard - can any of you confirm this this test is removed on purpose :)?